### PR TITLE
Fix tpm device permission issue for the dac nfs related testcases

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

# Format of PR title < sub-system: summary >
e.g
*virsh_migrate: Fix unsupported direct socket mode issue*

# Check lists by category
## If the PR is new cases
- [ ] Description of the cases
- [ ] Links of libvirt features, libvirt bugs or case IDs
- [ ] Test results

## If the PR is bug cases
- [ ] Bug descriptions or bug links
- [ ] Test results

## If the PR is a trivial fix
It it is the fix of typos, comments or documents, the description of PR could be skipped.

### CODE REVIEW

**Comment by yafu-1** [comment]
Test result:
$avocado run dac_nfs_save_restore..dynamic_ownership_off --vt-type libvirt
No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : 046307123d00894df0548d4e5c3592ab32fd3c90
JOB LOG    : /var/lib/avocado/job-results/job-2022-09-14T21.48-0463071/job.log
 (01/30) type_specific.io-github-autotest-libvirt.dac_nfs_save_restore.positive_test.exist_file.nfs_file_root.qemu_user.dynamic_ownership_off.no_root_squash: STARTED
 (01/30) type_specific.io-github-autotest-libvirt.dac_nfs_save_restore.positive_test.exist_file.nfs_file_root.qemu_user.dynamic_ownership_off.no_root_squash: PASS (9.08 s)
...
RESULTS    : PASS 30 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2022-09-14T21.48-0463071/results.html
JOB TIME   : 399.13 s

$avocado run dac_nfs_disk..dynamic_ownership_off --vt-type libvirt
No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : 1166ddd44fbc2862c7faf6d46427ea9ab2e36df7
JOB LOG    : /var/lib/avocado/job-results/job-2022-09-14T03.58-1166ddd/job.log
 (01/24) type_specific.io-github-autotest-libvirt.dac_nfs_disk.positive_test.nfs_img_root.root_user.dynamic_ownership_off.no_root_squash: STARTED
 (01/24) type_specific.io-github-autotest-libvirt.dac_nfs_disk.positive_test.nfs_img_root.root_user.dynamic_ownership_off.no_root_squash: PASS (44.71 s)
  ...
RESULTS    : PASS 24 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2022-09-14T03.58-1166ddd/results.html
JOB TIME   : 620.45 s

File: libvirt/tests/src/svirt/dac_nfs_disk.py [hunk 1]
```diff
@@ -40,6 +40,17 @@ def check_ownership(file_path):
     return label
 
 
+def set_tpm_perms(swtpm_lib):
```
Comment (chunfuwen) [inline libvirt/tests/src/svirt/dac_nfs_disk.py:43]: Can those logic move into avocado-vt as common function specific to tpm? maybe ./virttest/utils_libvirt/libvirt_tpm.py

Comment (yafu-1) [reply libvirt/tests/src/svirt/dac_nfs_disk.py:43]: The function is only needed in case of dynamic_ownership disabled, which is not a common use case.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by chloerh — APPROVED** [review]

**Pull request merged**